src/module.cpp

```diff
@@ -128,3 +130,22 @@ void Module::VerifyFunction() {
 		}
 	}
 }
+
+std::string Module::MapSpecieReact(const std::pair<std::string, int> &specie) {
+	return MapSpecieCoef(specie.second) + MapSpecie(specie.first) + " + ";
+}
+
+std::string Module::MapSpecieCoef(int coeff) {
+	return (coeff != 1) ? std::to_string(coeff) : "";
+}
+
+std::string Module::MapSpecie(const std::string &specie) {
+	return (name == constants::MAIN_MODULE) ? specie : name + "_" + specie;
```

Comment:
So I just realised something. `Compile()`is only ever really called on the main module. So this function might actually be completely unnecessary.